tailwind.config.js

```diff
@@ -0,0 +1,28 @@
+/** @type {import('tailwindcss').Config} */
```

Comment:
Can common colors be added here as well? so that way if we want to change the main font color or the main background color we can go to one file?

Reply:
Yep! I can add some variables and change things.